utils/lib/rr_graph/channel.py

```diff
@@ -243,318 +268,516 @@ def __str__(self):
 
 
 # Nice short alias..
-C = Channel
+T = Track
 
 
 class ChannelGrid(dict):
+    '''
+    Functionality:
```

Comment:
FYI - Docstring comments should be [reStructuredText](http://docutils.sourceforge.net/docs/ref/rst/introduction.html). 

If you write mostly markdown style it'll work okay.

Reply:
I'm not paying any attention to docstring stuff right now. I'll sort that out later